int32  lidarSpeed
int32  filterLev
---
int32  result